src/main/kotlin/uk/gov/gdx/datashare/controllers/EventsController.kt

```diff
@@ -184,6 +183,8 @@ class EventsController(
       ),
     ],
   )
+  @Timed("time.eventsController.getEvents", percentiles = [0.5, 0.90, 0.99])
+  @Counted("count.eventsController.getEvents")
```

Comment:
What does this do that we don't get already?

Reply:
My plan is to move to this style of doing our count metrics instead of using the meter registries and incrementing etc as we will be using the `@Timed` annotation anyways.

Reply:
Removing @TImed